src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,29 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
```

Comment:
The 'hidden' property should not be part of the Animal class constructor. Only herbivores should have the 'hidden' property. Consider removing it from the Animal class and initializing it directly within the Herbivore class.